src/main/java/org/spongepowered/api/event/SpongeEventFactory.java

```diff
@@ -171,17 +175,21 @@
 
     private static final FactoryProvider factoryProvider;
     private static final LoadingCache<Class<?>, EventFactory<?>> factories;
+    private static final Deque<EventFactoryPlugin> plugins;
 
     static {
         factoryProvider = new ClassGeneratorProvider("org.spongepowered.api.event.impl");
```

Comment:
Would it be preferable to put a marker interface in the target package and then use `Class.getPackage()` just to make this package declaration "soft"?